xbmc/guilib/DirtyRegion.h

```diff
@@ -35,3 +35,14 @@ class CDirtyRegion : public CRect
 };
 
 typedef std::vector<CDirtyRegion> CDirtyRegionList;
+
+class CGUIRenderInfo
+{
+public:
+  CGUIRenderInfo() { Reset(0); }
+  void Reset(unsigned int t) { currentTime = t; dirtyregions.clear(); visible = 0; total = 0; }
+  unsigned int currentTime;
+  CDirtyRegionList dirtyregions;
+  int visible;
+  int total;
```

Comment:
Using classes with public members  is never good design. Make it a struct if you don't need/want data encapsulation. This is c++ basics and no bike shedding, imo.